pkg/meta/version.go

```diff
@@ -2,7 +2,7 @@ package meta
 
 const (
 	// CLIAPIVersion used to communicate with user e.g. longhorn-manager
-	CLIAPIVersion    = 5
+	CLIAPIVersion    = 6
```

Comment:
Because we bump the CLI version, need to update the engine upgrade test image after merge. Chris will help with that.

Reply:
cc @longhorn/qa